amodem/__main__.py

```diff
@@ -234,9 +235,12 @@ def interface_factory():
         import pylab  # pylint: disable=import-error,import-outside-toplevel
         args.pylab = pylab
 
-    if args.audio_library == 'ALSA':
+    if args.audio_library == 'alsa':
```

Comment:
Can we keep the old value (for compatibility)?

Reply:
We can but the ALSA option wasn't even documented before...